Sources/MapboxCoreNavigation/EHorizon/RoadGraph.swift

```diff
@@ -32,6 +32,35 @@ public final class RoadGraph {
         }
         return LineString(locations.map { $0.coordinate })
     }
+    
+    /**
+     Returns a line string geometry corresponding to the given path.
+     
+     - returns: A line string corresponding to the given path, or `nil` if any of path edges is inaccessible.
```

Comment:
Maybe it is just me, but I don't understand what `inaccessible` means in this context.

Reply:
This method allows an arbitrary path to be passed in, so it probably refers to the case where the application passes in a long-irrelevant path that’s no longer connected to the current electronic horizon.

Reply:
We can add something along these lines to the documentation comment.

Reply:
<img width="655" alt="image" src="https://user-images.githubusercontent.com/1976216/124479711-34d01700-ddaf-11eb-9c5c-83f889bf8142.png">
This is a comment from the corresponding NavNative method, the folks say that this situation can happen if the navigation tile not in the cache anymore, I updated the comment accordingly.